client/my-sites/upgrades/checkout-thank-you/footer/index.jsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { isJetpackPlan } from 'lib/products-values';
+
+const CheckoutThankYouFooter = React.createClass( {
+	propTypes: {
+		isDataLoaded: React.PropTypes.bool.isRequired,
+		receipt: React.PropTypes.object.isRequired
```

Comment:
Something I just realized: we are supposed to render a generic version of the thank you page at `/checkout/:site/thank-you`, to handle the case where the API didn't return a receipt ID. This is broken in master, but this component will need to work without a `receipt` at some point.

I opened an issue for this here: https://github.com/Automattic/wp-calypso/issues/3607